packages/dev/src/common/Page.tsx

```diff
@@ -68,9 +69,11 @@ const AppToolbar = () => {
               variant={ButtonVariant.plain}
               onClick={() => qsContext.setActiveQuickStart(qsId)}
             >
-              <LightbulbIcon
-                color={qsContext.activeQuickStartID === qsId ? 'yellow' : 'currentColor'}
-              />
+              <Icon size="lg">
+                <LightbulbIcon
+                  color={qsContext.activeQuickStartID === qsId ? 'yellow' : 'currentColor'}
+                />
+              </Icon>
             </Button>
```

Comment:
This probably doesn't matter, but here's what the toolbar could look like with a few small tweaks.
* Remove `isFullHeight isStatic` from `<Toolbar>` - neither should be needed
* Remove `variant="icon-button-group"` from the `<ToolbarGroup>` that wraps these buttons. Since this has mixed content, a regular toolbar group probably looks better?
* Remove `spacer` from the `<ToolbarGroup>` - prop was removed, spacing is controlled via gap/columnGap/rowGap on a parent with children
* Change the `align` prop value from "alignRight" to "alignEnd"

Reply:
When I try to change the `align` prop, I get this error: `Type '"alignEnd"' is not assignable to type '"alignRight" | "alignLeft"'`
(I also saw the codemods try to change this prop, and I got this same error back then and created this issue for it: https://github.com/patternfly/pf-codemods/issues/646)

Reply:
d'oh! I'm sorry, just realized this PR is on react-core `6.0.0-alpha.50` and this change to the `align` prop was added in `6.0.0-alpha.57` via https://github.com/patternfly/patternfly-react/pull/10366.

Reply:
Latest version of alphas allow me to use `alignEnd` now. :)